examples/ocp_on_aws/aws_static_data.yml

```diff
@@ -1,8 +1,7 @@
 ---
 generators:
   - EC2Generator:
-      start_date: 2019-04-01
-      end_date: 2019-07-31
+      start_date: today
```

Comment:
I need to undo these changes to not break Andrew's new script in koku. I think that script will provide a better example of how to do this anyway.